### PR TITLE
HDXDSYS-705 misc schema changes

### PR DESCRIPTION
**Pull request by b-j-mills**
All pipelines are working and tests are passing!

I worry that moving the commit command in the operational presence pipeline into the loop so every entry is committed is slowing the pipeline down.  I ran it locally and it took 45 minutes.  I added a TODO to that part of the code in case anyone has a solution in mind.

### CODE REVIEW

File: src/hapi/pipelines/database/food_security.py [hunk 1]
```diff
@@ -84,34 +82,56 @@ def populate(self):
                         )
                         # Total population required to calculate fraction in phase
                         population_total = int(
-                            values[population_in_phase_columns["all"]][
+                            values[population_in_phase_columns["*"]][
                                 admin_code
                             ][irow]
                         )
-                        for ipc_phase_code in self._ipc_phase.data:
+                        dict_of_lists_add(
```
Comment (turnerm) [inline src/hapi/pipelines/database/food_security.py:89]: Why is this change needed? Can you add a comment explaining?

File: src/hapi/pipelines/database/humanitarian_needs.py [hunk 2]
```diff
@@ -90,18 +74,95 @@ def match_column(col, pattern_to_code):
...
+def _get_disabled_marker(col: Column) -> str:
+    disabled_marker = TagPattern.parse("#*+disabled").match(col)
+    if disabled_marker:
+        return "y"
+    if not disabled_marker:
+        return "*"
+
+
+# TODO: this is duplicate code, either move to shared location or overwrite with new HNO pipeline
+def _get_min_and_max_age(age_range: str) -> (int | None, int | None):
```
Comment (turnerm) [inline src/hapi/pipelines/database/humanitarian_needs.py:157]: Could it be moved to a utils file?

File: src/hapi/pipelines/database/humanitarian_needs.py [hunk 3]
```diff
@@ -90,18 +74,95 @@ def match_column(col, pattern_to_code):
...
+        TagPattern.parse("#*+returnees"): "RET",
+        TagPattern.parse("#*+idps"): "IDP",
+    }
+    population_group = match_column(col, population_group_patterns)
+    if not population_group:
+        population_group = "*"
+    return population_group
+
+
+def _get_age_range(hxl_tag: str) -> str:
```
Comment (turnerm) [inline src/hapi/pipelines/database/humanitarian_needs.py:135]: Is this also duplicate?

File: src/hapi/pipelines/database/operational_presence.py [hunk 4]
```diff
@@ -168,37 +161,36 @@ def populate(self, debug=False):
...
+                            org_acronym=org_acronym,
+                            org_name=org_name,
                             sector_code=sector_code,
                             reference_period_start=time_period_start,
-                            # TODO: Add to scraper (HAPI-199)
-                            source_data="not yet implemented",
                         )
                         self._session.add(operational_presence_row)
+                        # TODO: move this commit out of the loop once you figure out why it needs to be here
+                        self._session.commit()
```
Comment (turnerm) [inline src/hapi/pipelines/database/operational_presence.py:186]: What if you run it each time you add an entry to the org table? Or did you get a chance to try your own suggestion - populating orgs all at once with a dict? 

(If you want to do bulk updates - [here](https://github.com/OCHA-DAP/hdx-scraper-who/blob/feature/who.py#L251-L281) is some code I wrote recently to do so)

**Review by turnerm — APPROVED** [review]
So much great work Briar. Just a couple of small comments, and I think once you fix the 3W commit issue it should be ready to merge. And also update the changelog :)

**Review by mcarans — APPROVED** [review]
Well done for getting the remaining tests working. I think we can revisit the 3W issue in a separate PR.

File: src/hapi/pipelines/database/operational_presence.py (hunk 4 above)
Comment (b-j-mills) [reply src/hapi/pipelines/database/operational_presence.py:186]: I didn't get a chance to try populating all of the orgs at once, but that's next!  I did try running each time I added a new org, but the issue was still there.  It's almost like it's trying to commit the operational presence rows to the org table.
